pkg/apis/hive/v1alpha1/validating-webhooks/selector_syncset_validating_admission_hook_test.go

```diff
@@ -317,3 +541,11 @@ func testSelectorSyncSet() *hivev1.SelectorSyncSet {
 		},
 	}
 }
+
+func testSelectorSyncSetWithResources(resources ...runtime.Object) *hivev1.SelectorSyncSet {
+	ss := testSelectorSyncSet()
+	for _, resource := range resources {
+		ss.Spec.Resources = append(ss.Spec.Resources, runtime.RawExtension{Object: resource})
```

Comment:
Using a real object here requires the API type to be registered and the code for the type to be vendored. Instead, use raw yaml, which is what is being handled by the web hook anyway. This will also allow you to test for yaml that cannot be unmarshaled.